src/Resolvers/DomainTenantResolver.php

```
@@ -58,7 +59,19 @@ public static function findTenantByDomain(string $domain): (Tenant&Model)|null
 
     public static function isSubdomain(string $domain): bool
     {
-        return Str::endsWith($domain, config('tenancy.identification.central_domains'));
+        $centralDomains = Arr::wrap(config('tenancy.identification.central_domains'));
+
+        foreach ($centralDomains as $centralDomain) {
+            if ($domain === $centralDomain) {
+                return false;
+            }
+
+            if (Str::endsWith($domain, '.' . $centralDomain)) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Str' in method 'isSubdomain'. (undefined)

(StaticAccess)